dev/cvo-disable-operator.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterVersion
+metadata:
+  namespace: openshift-cluster-version
+  name: version
+spec:
+  overrides:
+  # this is my resource, but setting this override will not delete the operator, will it?
+  - kind: Deployment
+    # group: console.openshift.io
+    name: console-operator
+    namespace: openshift-console
+    unmanaged: true
+
+
+
+
+
+
```

Comment:
style nit: lots of extra newlines

Reply:
Oop, my editor isn't cleaning that up.  Let me fix.